ouroboros-consensus/src/Ouroboros/Consensus/HardFork/Combinator/Degenerate.hs

```diff
@@ -10,9 +10,10 @@
 
 {-# OPTIONS_GHC -Wno-orphans #-}
 
-module Ouroboros.Consensus.HardFork.Combinator.Degenerate (
```

Comment:
Our `StyleGuide.md` is to leave the paren where it was.

(IIUC `stylish-haskell` on the CI will enforce this, FYI.)